src/engine/controls/bpmcontrol.h

```diff
@@ -145,6 +146,9 @@ class BpmControl : public EngineControl {
     ControlPushButton* m_pTranslateBeatsLater;
     ControlEncoder* m_pTranslateBeatsMove;
 
+    // Locking/Unlocking BPM
+    std::unique_ptr<ControlPushButton> m_pToggleBpmLock;
```

Comment:
IMHO we need comments only if var names are not selfexplanatory, but I don't care if you keep it.

Reply:
Good points, thank you!